scripts/add_asimp_footer.py

```
@@ -0,0 +1,75 @@
+    "DBP-standard Bahasa Melayu Malaysia (Piawai) | "
+    "GNU General Public License v3.0"
+)
+
+
+def patch_markdown_file(filepath: str) -> None:
+    """Check a markdown file and append the standard ASIMP footer if absent.
+
+    Args:
+        filepath: The path of the markdown file to process.
+    """
+    with open(filepath, "r", encoding="utf-8") as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)